tests/jballerina-unit-test/src/test/resources/test-src/types/future/future_positive.bal

```diff
@@ -0,0 +1,180 @@
+function testBasicTypes() {
```

Comment:
Missing the licence header. Let's update it in a separate PR.

Reply:
Done